data.py-dist

```diff
@@ -26,6 +26,15 @@ PXE_CONFIG_SERVER = 'pxe'
 # Default VM images location
 DEF_VM_URL = 'http://pxe/images/'
 
+# Default shared ISO SR
```

Comment:
Why using another ISO SR when we already have `DEFAULT_NFS_ISO_DEVICE_CONFIG` in the data.py?

Reply:
Ah, I had missed that one

Reply:
In fact those are different, `DEFAULT_NFS_ISO_DEVICE_CONFIG` is used in tests that create a new ISO SR from an existing NFS server, while what I'm doing is using an existing ISO SR.

Using an existing shared ISO SR is in fact not perfect because of the possibility of interference with other users (filename clashes), but turning an existing NFS share into an ISO SR does not help this, and does not seem really suitable when working with hosts that already have an ISO SR configured.

OTOH the test could provide its own ISO SR to workaround the filename clashes - or, more comfortably, make a clever use of `mkstemp` through `ssh`.

Reply:
This was changed recently: we now use the master's ISO-SR mountpoint, no need for an extra config.